src/book/02-system/01-foundations/05-mathematics.mdx

```diff
@@ -207,6 +207,10 @@ If a velocity is multiplied with **time**, it becomes a **vector**.
 Eigen::Vector3d rA_tAb = vAb * t;
 ```
 
+## Units
+
+[S.I. units](https://en.wikipedia.org/wiki/International_System_of_Units) are used for everything but time. Storage of values may have a different unit, e.g. [nbs and nbs index files](https://nubook.nubots.net/system/foundations/nuclear#nbs-files).
```

Comment:
Thinking about these internal URLs, it's best to not include the domain, so the links still work if the domain is different (e.g. for previews, and for local builds).

```suggestion
[S.I. units](https://en.wikipedia.org/wiki/International_System_of_Units) are used for everything but time. Storage of values may have a different unit, e.g. [nbs and nbs index files](/system/foundations/nuclear#nbs-files).
```